.github/workflows/ci.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         rustup default 1.53.0 # [ref:rust_1_53_0]
 
         # Build and test.
-        cargo build --locked --release --target x86_64-pc-windows-msvc
+        RUSTFLAGS='-C target-feature=+crt-static' cargo build --locked --release --target x86_64-pc-windows-msvc
```

Comment:
Would you mind using the long flag (`--codegen`) instead of `-C`? Just to be consistent with the other flags here and for extra readability. And then we should wrap the line since it's getting too long. How about:

```rust
        RUSTFLAGS='--codegen target-feature=+crt-static' cargo build \
          --locked \
          --release \
          --target x86_64-pc-windows-msvc
```

Reply:
sure, will do